doc/man1/flux-resource.rst

```diff
@@ -109,6 +109,56 @@ snippet for an existing named format may be generated with
 ``--format=get-config=NAME``.  See :man1:`flux-jobs` *OUTPUT FORMAT*
 section for a detailed description of this syntax.
 
+The field names that can be specified for the **status** and **drain**
+commands are:
+
+**state**
+   State of node(s): "online", "offline", "avail", "exclude", "drain",
+   "draining", "drained", "all"
+
+**nnodes**
+   The number of nodes in state.
+
+**ranks**
+   The ranks of the nodes in state.
+
+**nodelist**
+   The nodes in state.
+
+**timestamp**
+   If node(s) in drain/draining/drained state, timestamp of node(s)
+   set to drain.
+
+**reason**
+   If node(s) in drain/draining/drained state, reason node(s) set to
+   drain.
+
+The field names that can be specified for the **list** are:
+
+**state**
+   State of node(s): "up", "down", "allocated", "free", "all"
+
+**properties**
+   Properties associated with resources in state.
+
+**nnodes**
+   The number of nodes in state.
+
+**ncores**
+   The number of cores in state.
+
+**ngpus**
+   The number of gpus in state.
+
+**ranks**
+   The ranks of the nodes in state.
+
+**nodelist**
+   The nodes in state.
```

Comment:
These all have the same comments as above.

Since this is duplicated information, I wonder if there is some way to share the list between the `status` and `list` subcommands and maybe just explain the differences? (Just an idea, if that is awkward, ignore it)

Reply:
I had considered that approach, going with perhaps something like:

```
**nnodes**
   (list, status, drain) ....
   
**reason**
   (status, drain) ...   
```

but the `state` being so different between status and list, and them having a few fields that don't intersect, felt best to separate them out.